pipeline/alert.go

```diff
@@ -1016,9 +1016,9 @@ func (n *AlertNodeData) PagerDuty2() *PagerDuty2Handler {
 type PagerDuty2Handler struct {
 	*AlertNodeData `json:"-"`
 
-	// The service key to use for the alert.
+	// The routing key to use for the alert.
 	// Defaults to the value in the configuration if empty.
-	ServiceKey string `json:"serviceKey"`
+	RoutingKey string `json:"routingKey"`
 }
```

Comment:
If you add a `ServiceKey` function to the `PagerDuty2Handler`  that takes a string and sets the `RoutingKey` field then we maintain backwards compatibility.